### DISCUSSION TITLE
Reader Statistics: is there a way to combine multiple items

### DISCUSSION THREAD

**Discussion by mergen3107** [koreader/koreader#7889] [Q&A]
Adding to #7887, may I ask is there ability to combine multiple items' statistics under one title? 

For example, I would like to have reading time for all Volumes of one title adding up to a common reading time. 

It would be enough if I was able to just sum all reading times after I finished reading the Volumes.

Thank you!

**Comment by NiLuJe** [comment]
Not in the UI, no, but you can probably come up with something via SQL queries.

(There's an open/closed issue about that, IIRC).

**Reply by mergen3107** [reply]
Thank you!
I'll try something
